controlpanel/api/cluster.py

```diff
@@ -63,6 +73,12 @@ def create(self):
             f"--set=Username={self.user.slug}",
         )
 
+    def reset(self):
+        """
+        Reset the user's home directory.
+        """
+        helm.reset_home(f"init-user-{self.user.slug}")
```

Comment:
Why this is passing the `init-user-USER` helm release name?

We're not planning to re-use [the `init-user` helm chart](https://github.com/ministryofjustice/analytics-platform-helm-charts/tree/master/charts/init-user) right?

Reply:
PLaceholder (again)